src/cmd/gen_sign_initial_ds.cpp

```diff
@@ -23,10 +23,18 @@
 
 #include <boost/property_tree/ptree.hpp>
 #include <boost/property_tree/xml_parser.hpp>
+#include "boost/program_options.hpp"
```

Comment:
This program was inteded to be used with a single keypair. Any reason for changin?

Reply:
The motivation for all binaries in _cmd/_ subfolder was the same - more input params validation together with incorporation of _boost/prograp_options_.
If you consider parameter flags being decrease in "user friendliness", then I can revert these changes. From perspective of scripts, only the _release.sh_ uses it and I have already done the adaptation here, so it shouldn't be a problem.